### PR TITLE
Orleans v7

### PR DESCRIPTION
**Pull request by SebastianStehle**

### CODE REVIEW

File: Test/Grains/ReminderGrain.cs [hunk 1]
```diff
@@ -9,14 +9,14 @@ public sealed class ReminderGrain : Grain, IReminderGrain
     {
         public async Task RemoveReminder(string reminder)
         {
-            var reminderType = await GetReminder(reminder);
+            var reminderType = await this.GetReminder(reminder);
```
Comment (SebastianStehle) [inline Test/Grains/ReminderGrain.cs:12]: There are a lot of these "this." fixes. I am fine when you change the coding style later, but for now we should revert this to make the review process easier.

File: Test/Host/TestConfigurator.cs [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+using Orleans.Configuration;
+using Orleans.Hosting;
+using Orleans.Providers.MongoDB.Test.GrainInterfaces;
+using Orleans.TestingHost;
+using System;
+using System.Net;
+
+namespace Orleans.Providers.MongoDB.Test.Host
+{
+  internal class TestConfigurator : ISiloConfigurator
```
Comment (SebastianStehle) [inline Test/Host/TestConfigurator.cs:13]: Please change to 4 spaces.

File: Test/Grains/ReminderGrain.cs (hunk 1 above)
Comment (MarkusGeigerDev) [reply Test/Grains/ReminderGrain.cs:12]: It's not coding style. They changed it from a base class method to an extension method, so the "this" is actually necessary.

File: Test/Host/TestConfigurator.cs (hunk 2 above)
Comment (MarkusGeigerDev) [reply Test/Host/TestConfigurator.cs]: done

**Comment by SebastianStehle** [comment]
So far it LGTM, waiting for the submodule thing ;)

**Pull request closed without merging**